src/Test/Unit/Config/SchemaTest.php

```diff
@@ -77,6 +77,7 @@ public function testGetDefaultsForBuild(): void
                 BuildInterface::VAR_SCD_MAX_EXEC_TIME => null,
                 BuildInterface::VAR_ERROR_REPORT_DIR_NESTING_LEVEL => 1,
                 BuildInterface::VAR_SCD_USE_BALER => false,
+                BuildInterface::VAR_QUALITY_PATCHES => ''
```

Comment:
I believe you will have to bump the `magento-cloud-patches` dependency in `composer.json`

Reply:
It is for https://github.com/magento/quality-patches